### ISSUE TITLE
Empty titles in tree view after updating to 3.2

### ISSUE THREAD

**Issue by GertTimmerman** [statamic/cms#4406]
## Bug Description
I have updated statamic from 3.1 to 3.2. Locally everything works as excepted, but in production i don't see titles in the tree view of collections. In the XHR response of the tree i do see empty titles, the entry title field does contains the title

## How to Reproduce
Sorry, I really don't know

## Extra Detail
Empty titles in tree view:
![Empty Title](https://www.fritsjurgens.com/assets/dev/Schermafbeelding-2021-10-06-om-11.38.14.png)

Response of /cp/collections/pages/tree?site=default
![Response](https://www.fritsjurgens.com/assets/dev/Schermafbeelding-2021-10-06-om-12.05.58.png)

## Environment

Statamic 3.2.11 Pro
Laravel 8.63.0
PHP 7.4.22
4rn0/statamic-v3-image-optimizer 1.0.8
doublethreedigital/duplicator 2.0.0
gerttimmerman/statamic-alignment-fieldtype dev-master
gerttimmerman/statamic-cpbar dev-master
gerttimmerman/statamic-form-connector dev-master
joetannenbaum/antlers-stack-tags 0.1.0
optimoapps/statamic-bard-text-align 1.0.2
rias/statamic-button-box 2.0.3
spatie/statamic-algolia-places 1.1.0
spatie/statamic-responsive-images 2.8.0
withcandour/aardvark-seo 2.0.16

**Comment by duncanmcclean** [comment]
Try deleting your `vendor` directory and running `composer install`, it should fix your issue.

Closing as it's a duplicate of #4180.

**Comment by GertTimmerman** [comment]
> Try deleting your `vendor` directory and running `composer install`, it should fix your issue.
> 
> Closing as it's a duplicate of #4180.

This does not help for me .. still having empty titles

**Comment by duncanmcclean** [comment]
Have you hard refereshed? Tried a different browser? Tried running `php artisan statamic:install`?

**Comment by GertTimmerman** [comment]
> Have you hard refereshed? Tried a different browser? Tried running `php artisan statamic:install`?

What do you exactly mean with hard refresh? Hard refresh of the browser?

I have deleted all my browser data => does not help
I have tried other browsers => same result, empty titles
I have deleted all cache (php artisan optimize:clear & php please stache:clear) => does not help
I have runned php artisan statamic:install => does not help
I have deleted cache from cloudflare => does not help
I have disabled caching with cloudflare => does not help

**Comment by duncanmcclean** [comment]
Hmm, strange. Would have thought one of those would have fixed it. Maybe @jasonvarga will have some ideas when he's online.

**Comment by bwaltz** [comment]
I have also experienced this problem with the tree view in my Pages collection. I was able to start a fresh Peak install and the issue was resolved, but I understand this option won't feasible for everyone.

**Comment by francoismajor** [comment]
I don't know if this is related, but any tree picker (entries) shows blanks, or the default site name if I specify one.

I've tried all proposed solutions above and no change. (deleted root vendor, public/vendor, composer install, php artisan statamic:install)

- I have a multisite setup.
- My tree shows the name of the default 'site' for all pages, whether I select one site or the other. 
(I set up a name in sites.php : ` 'name' => env('APP_DEFAULT_LOCALE_NAME', config('app.name')),`
- Leaving the default config there gives blanks instead. `'name' => config('app.name'),` (which I would guess is the default setup for everyone reporting this)
- It does not pickup the APP_NAME setup in .env.

:arrow_down: These should be pages (Accueil and Demo)
<img width="609" alt="Screen Shot 2021-10-06 at 6 57 14 PM" src="https://user-images.githubusercontent.com/8515329/136294793-fc76f881-b795-4134-8fad-e3421c6ecd7d.png">
